staging/src/k8s.io/kubectl/pkg/cmd/exec/exec.go

```diff
@@ -326,10 +326,11 @@ func (p *ExecOptions) Run() error {
 	containerName := p.ContainerName
 	if len(containerName) == 0 {
 		if len(pod.Spec.Containers) > 1 {
-			fmt.Fprintf(p.ErrOut, "Defaulting container name to %s.\n", pod.Spec.Containers[0].Name)
 			if p.EnableSuggestedCmdUsage {
```

Comment:
`EnableSuggestedCmdUsage` checks if `kubectl describe` exists or not, if you really want to delete `Use describe` info, you should also delete all these `EnableSuggestedCmdUsage` logic.

Reply:
Roger, agree this.